### ISSUE TITLE
Why choose the subscript is 1? --x[:,:1]

### ISSUE THREAD

**Issue by ericyq** [Event-AHU/OpenPAR#16]
if args.use_div :
            x = torch.cat([x[:,:1],self.part_class_embedding.to(x.dtype) + torch.zeros(x.shape[0], 1, x.shape[-1], dtype=x.dtype, device=x.device)
                                ,x[:,1:]],dim=1)

**Comment by 1125178969** [comment]
is to insert the Part CLS Token between the G lobal CLS Tokn and the Visual Token

**Comment by ericyq** [comment]
Can the subscript number 1 become the last one?
or can the subscript 1 be arbitrary?

**Comment by 1125178969** [comment]
It is not possible to slice x in this way, because x[:,0] represents the CLS token
